go.mod

```diff
@@ -1,17 +1,73 @@
 module github.com/adevinta/go-testutils-toolkit
 
-go 1.22
+go 1.24.0
+
+toolchain go1.24.5
 
 require (
+	github.com/adevinta/go-k8s-toolkit v0.0.0-20240913131011-109953036918
+	github.com/adevinta/go-log-toolkit v0.0.0-20241010122356-50ef5e036d19
 	github.com/adevinta/go-system-toolkit v0.0.0-20240912143443-133d8c380cfc
+	github.com/cenkalti/backoff v2.2.1+incompatible
+	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/afero v1.8.2
-	github.com/stretchr/testify v1.7.1
-	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c
+	github.com/stretchr/testify v1.10.0
+	gopkg.in/yaml.v3 v3.0.1
+	k8s.io/api v0.33.3
+	k8s.io/apimachinery v0.33.3
+	sigs.k8s.io/controller-runtime v0.21.0
```

Comment:
addint those dependencies to a `test toolkit` is a smell.
This should probably be a distinct module